### PR TITLE
Update snap documentation

### PR DESCRIPTION
**Pull request by anonymouse64**
Fixes some of issue #670 (notably doesn't add documentation about how to use the snap with services, as that isn't done yet).

### CODE REVIEW

File: snap/README.md [hunk 1]
```diff
@@ -1,73 +1,179 @@
...
+```bash
+$ sudo snap install edgexfoundry --edge
+```
 
-`$ sudo snap install edgexfoundry --channel=california/edge`
+You can specify install specific releases using the `--channel` option. For example to install the california release of the snap:
 
-**Note** - this snap has only been tested on Ubuntu 16.04 LTS Desktop/Server and Ubuntu Core 16.
+```bash
+$ sudo snap install edgexfoundry --channel=california
```
Comment (tonyespy) [inline snap/README.md]: This won't work right now, as we never promoted to stable. Let's discuss this later today.

File: snap/README.md [hunk 2]
```diff
@@ -1,73 +1,179 @@
...
 
-`$ snap connect edgexfoundry-core:system-observe core:mount-observe`
+**Note** - these interface will be connected automatically after https://forum.snapcraft.io/t/edgexfoundry-auto-connections-assertions-request/7920/1 has been processed. After this is done, then connecting these interfaces will only be necessary during development. See the section below on building the snap for more details.
 
+**Note** - the process-control interface will be dropped after all of the services are supported as proper daemons in the snap.
 
-## Starting/Stopping EdgeX
-To start all the EdgeX microservices, use the following command:
+## Using the EdgeX snap
+To start all the EdgeX microservices, run the following command as root:
```
Comment (tonyespy) [inline snap/README.md]: I'd change to "as root" to "using sudo".

File: snap/README.md [hunk 3]
```diff
@@ -1,73 +1,179 @@
...
 
 `$ edgexfoundry.start-edgex`
 
-To stop all the EdgeX microservices, use the following command:
+To stop all the EdgeX microservices, run the following command as root:
```
Comment (tonyespy) [inline snap/README.md]: Same as above ("sudo").

File: snap/README.md [hunk 4]
```diff
@@ -1,73 +1,179 @@
...
+```
+
+After installing the snap, you will need to connect interfaces and restart the snap. To connect the interfaces manually, run:
+
+```bash
+$ sudo snap connect edgexfoundry:hardware-observe
+$ sudo snap connect edgexfoundry:process-control
+$ sudo snap connect edgexfoundry:system-observe
+$ sudo snap connect edgexfoundry:mount-observe
+```
```
Comment (tonyespy) [inline snap/README.md]: I would just refer to the section above, as this just duplicates it.

File: snap/README.md [hunk 5]
```diff
@@ -1,73 +1,179 @@
...
- * install snapcraft
- * clone this git repo
- * cd edgex-core-snap
- * snapcraft
+After connecting the interfaces, restart all of the snap's services with:
 
-This should produce a binary snap package called edgex-core-snap_<latest version>_<arch>.snap.
+```bash
+$ sudo snap restart edgexfoundry
+```
```
Comment (tonyespy) [inline snap/README.md]: Same as above (this has already been covered).

File: snap/README.md (hunk 2 above)
Comment (anonymouse64) [reply snap/README.md]: Done

File: snap/README.md (hunk 3 above)
Comment (anonymouse64) [reply snap/README.md]: Done

File: snap/README.md (hunk 1 above)
Comment (anonymouse64) [reply snap/README.md]: As discussed, this will just be changed to `delhi`.

**Pull request merged**